src/components/OSCALEditableTextField.test.js

```diff
@@ -8,6 +8,7 @@ test("OSCALEditableTextField loads", () => {
     <OSCALEditableTextField
       canEdit
       editedField={["version"]}
+      isInEditState={[false, () => {}]}
```

Comment:
This was something @mikeisen1 had questions about on @kperk103's recent PR. Please coordinate with @kperk103 and someone else who worked on that testing PR to see if this is the appropriate way to handle this

Reply:
We got rid of this in the fix for that PR, so it can be removed.